src/App.js

```diff
@@ -1,14 +1,23 @@
 import './App.css';
 import { React, useState, useEffect } from "react";
 import { Routes, Route, BrowserRouter } from "react-router-dom";
+import Header from "./components/header-components/Header.jsx";
+import Nav from "./components/header-components/Nav.jsx";
+import ArticleList from './components/article-components/ArticleList';
 
 function App() {
+
+
   return (
-    <div className="App">
-      <header className="App-header">
-        
-      </header>
+    <BrowserRouter>
+        <div className="App">
+          <Header />
+          <Nav />
+          <ArticleList />
```

Comment:
Might be good to have a home route '/' - You'll have to adjust for future routes anyway